Framework/Algorithms/src/MedianDetectorTest.cpp

```diff
@@ -246,25 +246,25 @@ int MedianDetectorTest::maskOutliers(
   }
 
   for (size_t i = 0; i < indexmap.size(); ++i) {
-    std::vector<size_t> hists = indexmap.at(i);
+    std::vector<size_t> hists = indexmap[i];
     double median = medianvec.at(i);
 
     PARALLEL_FOR1(countsWS)
     for (int j = 0; j < static_cast<int>(hists.size()); ++j) {
```

Comment:
Loop not converted, I don't see I reason why not? Maybe the cast confused the tool?